### PR TITLE
Bump version of grpc and protobuf 

### PR DESCRIPTION
**Pull request by guohao**

### CODE REVIEW

File: contrib/grpc-contrib/src/main/java/com/salesforce/grpc/contrib/StaticResolver.java [hunk 1]
```diff
@@ -154,8 +154,8 @@ private static class StaticResolverFactory extends NameResolver.Factory {
 
         @Nullable
         @Override
-        public NameResolver newNameResolver(URI targetUri, Attributes params) {
-            return provider.newNameResolver(targetUri, params);
+        public NameResolver newNameResolver(URI uri, NameResolver.Args args) {
```
Comment (nikolay-pshenichny) [inline contrib/grpc-contrib/src/main/java/com/salesforce/grpc/contrib/StaticResolver.java:157]: nit: it seems that the first param is still named `targetUri` in grpc::NameResolver (https://github.com/grpc/grpc-java/blame/f082151fb5de1c102a492e2fa2bc4c1bef0d8c6a/api/src/main/java/io/grpc/NameResolver.java#LL151C34-L151C49)
and other places in this PR keep the same naming. Will it be possible to revert this rename?

Comment (guohao) [reply contrib/grpc-contrib/src/main/java/com/salesforce/grpc/contrib/StaticResolver.java:157]: sure

**Review by nikolay-pshenichny — APPROVED** [review]

**Comment by rmichela** [comment]
Close/Reopen to bounce CI

**Comment by nikolay-pshenichny** [comment]
@guohao, could you please take a look at the build failures? Thank you!

**Comment by guohao** [comment]
> @guohao, could you please take a look at the build failures? Thank you!

Trying to fix it

**Comment by guohao** [comment]
@rmichela @nikolay-pshenichny Tests have passed :D

File: contrib/grpc-spring/pom.xml [hunk 2]
```diff
@@ -42,7 +42,6 @@
             <artifactId>servlet-api</artifactId>
             <version>2.5</version>
             <scope>provided</scope>
-            <optional>true</optional>
```
[the remark this answers is not in the document]
Comment (guohao) [reply contrib/grpc-spring/pom.xml]: This line has been added back

**Pull request merged**